packages/core/src/dialog/DialogHeader.css

```diff
@@ -10,6 +10,7 @@
 }
 .saltDialogHeader-header {
   margin: 0;
+  overflow-wrap: anywhere;
```

Comment:
should this be in this pr/does it need cs?

Reply:
> should this be in this pr/does it need cs?

Ah, shouldn't be included